library/src/webapp-filtered/editor/ckeditor.launch.js

```diff
@@ -351,7 +357,7 @@ sakai.editor.editors.ckeditor.launch = function(targetId, config, w, h) {
         browserContextMenuOnCtrl: true,
         
         // Fix the smileys to a single location
-        smiley_path: `/library/webjars/ckeditor4/${ckeditor.version}/plugins/smiley/images/`,
+        smiley_path: `/library/webjars/ckeditor4/4.21.0/plugins/smiley/images/`,
```

Comment:
Should all those versions be hardcoded?

Reply:
No! Thanks for spotting that. No idea what was going on there.